src/integrations/prefect-dbt/prefect_dbt/cli/commands.py

```diff
@@ -789,3 +828,106 @@ def create_summary_markdown(results: dbtRunnerResult, command: str) -> UUID:
+```
+
+"""
+
+
+def create_unsuccessful_markdown(results: dbtRunnerResult, command: str) -> UUID:
+    """
+    Creates a Prefect task markdown artifact summarizing the results
+    of unsuccessful nodes, including compiled code.
+    """
+    if results.exception is None:
+        failed_nodes = []
+        successful_nodes = []
+        errored_nodes = []
+        skipped_nodes = []
+        warned_nodes = []
+        markdown = """"""
+        for r in results.result.results:
+            if r.status == NodeStatus.Fail:
+                failed_nodes.append(r)
+            elif r.status == NodeStatus.Error:
+                errored_nodes.append(r)
+            elif r.status == NodeStatus.Skipped:
+                skipped_nodes.append(r)
+            elif r.status == NodeStatus.Success:
+                successful_nodes.append(r)
+            elif r.status == NodeStatus.Warn:
+                warned_nodes.append(r)
+
+        markdown += f"""# dbt {command} Task Unsuccessful
+
+| Successes | Errors | Failures | Skips | Warnings |
+| :-------: | :----: | :------: | :---: | :------: |
+| {len(successful_nodes)} |  {len(errored_nodes)} | {len(failed_nodes)} | {len(skipped_nodes)} | {len(warned_nodes)} |
+
+----
+
+"""
+        if len(errored_nodes) > 0:
+            markdown += "\n## Errored Nodes:\n"
+            for n in errored_nodes:
+                markdown += _create_node_info_md(
+                    n.node.name,
+                    n.node.resource_type,
+                    n.message,
+                    n.node.path,
+                    n.node.compiled_code,
+                )
+        if len(failed_nodes) > 0:
+            markdown += "\n## Failed Nodes:\n"
+            for n in failed_nodes:
+                markdown += _create_node_info_md(
+                    n.node.name,
+                    n.node.resource_type,
+                    n.message,
+                    n.node.path,
+                    n.node.compiled_code,
+                )
+        if len(skipped_nodes) > 0:
+            markdown += "\n## Skipped Nodes:\n"
+            for n in skipped_nodes:
+                markdown += _create_node_info_md(
+                    n.node.name,
+                    n.node.resource_type,
+                    n.message,
+                    n.node.path,
+                    n.node.compiled_code,
+                )
+        if len(warned_nodes) > 0:
+            markdown += "\n## Warned Nodes:\n"
+            for n in warned_nodes:
+                markdown += _create_node_info_md(
+                    n.node.name,
+                    n.node.resource_type,
+                    n.message,
+                    n.node.path,
+                    n.node.compiled_code,
+                )
+        return markdown
+    elif not results.success and results.exception is not None:
+        markdown = f"""## dbt {command} Task Unsuccessful
+
+Run results contained the following exception: {results.exception}"""
+        return markdown
```

Comment:
totally not important for this PR, but it feels like it might be nice to do this python objects -> `str` with a jinja template

Reply:
I like that idea